tests/acceptance/features/apiSearch1/search.feature

```diff
@@ -291,3 +291,127 @@ Feature: Search
     When user "Alice" searches for 'AND mediatype:document' using the WebDAV API
     Then the HTTP status code should be "400"
     And the value of the item "/d:error/s:message" in the response should be "error: bad request: the expression can't begin from a binary operator: 'AND'"
+
+
+  Scenario Outline: search a file globally (in all drives)
```

Comment:
i think we can extend tests for different pattern

Reply:
IMHO, testing for different patterns in this case would be repetitive, would't it?